docs/cs-tools/scriptability/README.md

```diff
@@ -0,0 +1,90 @@
+# Searchable Content
```

Comment:
did you run `mkdocs serve` or `scripts/develop-docs.sh` and view your documentation locally? (i totally didn't show you how to do this yet)

if you have and you're happy, then I'm happy. do you plan to write a KB article for this one too?

Reply:
Yes.  Great to know.